# This patch developed inhouse fixes some Solaris specific issues
# and has not been submitted to the community.
--- a/src/cmd/builtin/asa.c	2014-07-14 00:38:14.796528534 -0700
+++ b/src/cmd/builtin/asa.c	2014-07-14 00:38:58.469687039 -0700
@@ -111,7 +111,7 @@
 	register Sfio_t	*fp;
 	register int	n, reclen=0;
 
-	cmdinit(argc, argv, (void*)0, (const char*)0, 0);
+	cmdinit(argc, argv, context, ERROR_CATALOG, 0);
 	while (n = optget(argv, usage)) switch (n)
 	{
 	    case 'r':
